src/controllers/StudentController.js

```diff
@@ -35,23 +35,24 @@ module.exports = {
       const student = await Student.findByPk(req.params.id);
       if (student) {
         const updatedStudent = await student.update({
```

Comment:
```suggestion
        const updatedStudent = await student.update(req.body);
```
можно просто req.body передать